### ISSUE TITLE
Missing dependency Text::Diff and failing test with 0.85

### ISSUE THREAD

**Issue by perlpunk** [ingydotnet/inline-pm#76]
While maintaining the perl modules repository in SUSE Open Build Service, I noticed a problem with Inline.pm:
https://build.opensuse.org/package/live_build_log/devel:languages:perl:autoupdate/perl-Inline/standard/x86_64

```
[   16s] #   Failed test 'Test Inline->derive_minus_I -- TEST 1 - No settings'
[   16s] #          got: '../../../../../usr/lib/perl5/site_perl
[   16s] # '
[   16s] #     expected: ''
[   16s] Can't locate Text/Diff.pm in @INC (you may need to install the Text::Diff module) (@INC contains: t inc/lib /home/abuild/rpmbuild/BUILD/Inline-0.85/blib/lib /home/abuild/rpmbuild/BUILD/Inline-0.85/blib/arch /usr/lib/perl5/site_perl/5.30.1/x86_64-linux-thread-multi /usr/lib/perl5/site_perl/5.30.1 /usr/lib/perl5/vendor_perl/5.30.1/x86_64-linux-thread-multi /usr/lib/perl5/vendor_perl/5.30.1 /usr/lib/perl5/5.30.1/x86_64-linux-thread-multi /usr/lib/perl5/5.30.1 /usr/lib/perl5/site_perl .) at inc/lib/TestML/Run/TAP.pm line 45, <INPUT> line 1.
[   16s] t/09perl5lib.t ........... 
```
Text::Diff is not listed as a dependency.

**Comment by perlpunk** [comment]
The output for 0.85_002:
```
[   19s] PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
[   19s] t/000-require-modules.t .. ok
[   20s] t/01usages.t ............. ok
[   20s] t/02config.t ............. ok
[   20s] t/03errors.t ............. ok
[   21s] t/04create.t ............. ok
[   21s] Skipping - couldn't load the Inline::Files module
[   21s] t/05files.t .............. ok
[   22s] t/06rewrite_config.t ..... ok
[   22s] t/07rewrite2_config.t .... ok
[   22s] t/08unicode.t ............ ok
[   23s] #   Failed test 'Test Inline->derive_minus_I -- TEST 1 - No settings'
[   23s] #          got: '../../../../../usr/lib/perl5/site_perl
[   23s] # '
[   23s] #     expected: ''
[   23s] #   Failed test 'Test Inline->derive_minus_I -- TEST 2 - Add relative paths to @INC'
[   23s] #          got: '../../../../../usr/lib/perl5/site_perl
[   23s] # doc
[   23s] # eg
[   23s] # '
[   23s] #     expected: 'doc
[   23s] # eg
[   23s] # '
[   23s] #   Failed test 'Test Inline->derive_minus_I -- TEST 3 - Non-existing paths are removed'
[   23s] #          got: '../../../../../usr/lib/perl5/site_perl
[   23s] # doc
[   23s] # eg
[   23s] # '
[   23s] #     expected: 'doc
[   23s] # eg
[   23s] # '
[   23s] t/09perl5lib.t ........... 
[   23s] Failed 3/3 subtests 
[   23s] t/author-pod-syntax.t .... skipped: these tests are for testing by the author
[   23s] 
[   23s] Test Summary Report
[   23s] -------------------
[   23s] t/09perl5lib.t         (Wstat: 0 Tests: 3 Failed: 3)
[   23s]   Failed tests:  1-3
[   23s] Files=11, Tests=33,  4 wallclock secs ( 0.06 usr  0.04 sys +  3.35 cusr  0.35 csys =  3.80 CPU)
[   23s] Result: FAIL
[   23s] Failed 1/11 test programs. 3/33 subtests failed.
```

**Comment by perlpunk** [comment]
fixed in 0.86
